runtime/src/main/java/org/corfudb/runtime/object/transactions/OptimisticTransactionalContext.java

```diff
@@ -88,15 +88,17 @@
             // If we don't own this object, roll it back
             // to avoid interrupting short-lived micro TX, grab the lock
             if (oCtxt != null && oCtxt != this) {
+                /* */
```

Comment:
added a comment to explain: This is the block that reduces contention, and also, potentially, concurrency.